packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -37,61 +37,44 @@ const trace = makeTracer(' VM ');
 // some number of outstanding loans, each called a Vault, for which the
 // collateral is provided in exchange for borrowed RUN.
 
-/** @type {MakeVaultManager} */
+/**
+ * @param {ContractFacet} zcf
+ * @param {ZCFMint} runMint
+ * @param {Brand} collateralBrand
+ * @param {ERef<PriceAuthority>} priceAuthority
+ * @param {{
+ *  'ChargingPeriod': ParamRecord<'relativeTime'> & { value: RelativeTime },
+ *  'RecordingPeriod': ParamRecord<'relativeTime'> & { value: RelativeTime },
```

Comment:
again: why quote these keys?

```suggestion
 *  ChargingPeriod: ParamRecord<'relativeTime'> & { value: RelativeTime },
 *  RecordingPeriod: ParamRecord<'relativeTime'> & { value: RelativeTime },
```

Reply:
vestiges of refactoring I guess. will remove.

Reply:
Doesn't seem to have been cleaned up before merge?

Reply:
mea culpa. I'll include in another PR